object/slicer/slicer.go

```diff
@@ -263,7 +263,7 @@ func initPayloadStream(ctx context.Context, ow ObjectWriter, header object.Objec
 	} else if opts.bearerToken != nil {
 		prm.WithBearerToken(*opts.bearerToken)
 		// token issuer is a container owner.
-		issuer := opts.bearerToken.ResolveIssuer()
+		issuer := opts.bearerToken.Issuer()
```

Comment:
This one as well.

Reply:
nope as well, this will work correctly only with `SignByIssuer` usage introduced in current commit

Reply:
one commit after https://github.com/nspcc-dev/neofs-sdk-go/pull/564#discussion_r1513048024